### ISSUE TITLE
[0.7.0] (Almost entirely) black screen

### ISSUE THREAD

**Issue by williamjcm** [Backdash/MonikaModDev#853]
So, what happened is a little strange. I did an in-game update from 0.6.3 to 0.7.0 five minutes ago, and I deleted my MAS persistent save to start fresh.

I then launched DDLC, went to the settings menu, and disabled room animations.

What happened is that the menu closed and the after story started, but I saw a black screen where the room and Monika should have been. The GUI was still visible though.

EDIT: I completely forgot to add that toggling room animations "works" (the checkbox gets toggled), but the closing menu happens every time, and the black screen seems permanent.

EDIT 2: I just started fresh, and the issue still happened. It seems the game can't initialize the background, because Monika becomes visible after the intro, but the background stays black. I'll try deleting my persistent save again and start the After Story normally, and only disable animations after the game begins.

EDIT 3: here's a picture of what happens:
![screenshot0001](https://user-images.githubusercontent.com/13572456/35513310-878855ca-04fa-11e8-9550-86ee16cce668.png)

**Comment by therationalpi** [comment]
I'll try to figure out what happened on this. Thanks for the bug report.

**Comment by williamjcm** [comment]
You're welcome.

I'd like to add that just I experienced something similar after starting fresh.

I ran the in-game updater just in case, and clicking cancel on the "no updates at this time" dialog also ended up launching the game, bypassing the name screen. However, I didn't end up with a black screen like above.

**Comment by MendicantMieu** [comment]
Hello and good day to you!

I wanted to say that I also had this issue as well as one other that may coincide with this bug.

When I first launched the game, I chose "No" for my first time.  Of course, Monika says, "When did you have a choice?" and then her sprite vanishes and just zooms in on the background.  My second time around, the same thing.  So, I chose to do the "Disable Room Animations" and got the same issue.

I figured that it was just a small glitch, that when an image changed, it would redraw everything.  So, again ... I made the choice of saying "No" at the beginning.  Suddenly, I get the Monika Jumpscare (Chara) sprite on the screen, but then everything stayed black.

So, just for the hell of it ... I cleared all the data I could for the game.  (C:\Users ... Etc) as well as the base files.  Started from scratch again ... same issue with the missing jumpscare sprite.

The only thing I cannot remember is what mode I had it in, if it was windowed or full screen.  (As I said screw it, one little bug isn't going to stop me from enjoying the rest of your work.)

So yeah ... I figured that I would add this to the current ticket submitted here ... just in case anyone else has had or will have this issue.

Thank you again for your time in this matter ... and also thank you for your time creating this project.
